scripts/demo/snapshot.sh

```
@@ -80,9 +80,45 @@ serve() {
+  verify_pid=""
+  cleanup() { [ -n "${verify_pid:-}" ] && kill "$verify_pid" 2>/dev/null || true; }
+  trap cleanup EXIT
+
+  info "Serving the export on 127.0.0.1:${port} for verification ..."
+  node24 npx --yes serve "$out_dir" -l "$port" >/dev/null 2>&1 &
+  verify_pid=$!
+
+  # -q --http1.1: an --http2 line in the operator's curlrc makes this request an h2c
+  # upgrade that the server answers by closing the connection. Cost a full session once.
+  local ready=0 i
+  for i in $(seq 1 40); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)